debian/rules

```diff
@@ -14,7 +14,6 @@ override_dh_auto_test:
 override_dh_clistrip:
 
 override_dh_auto_build:
-	git submodule update --init
```

Comment:
this might still be needed if we ship `taglib` as submodule (or anything else for that matter), why drop?

Reply:
Afaik normally Debian packages get built from a directory where the source code is already checked out / prepared correctly, so this shouldn't be needed.

Reply:
Yea that's how it should happen. I added this to help new users before we got the repo set up, but if we do the building its fine to remove.